### PR TITLE
Feature/web/bigquery/integration tests

### PR DESCRIPTION
**Pull request by bill-warner**
Includes:
* Great Expectations `integration_test` suite
* `integration_test.sh` script which runs the web model 5 times over the test dataset, in 1 day increments. During each run, it checks the outputted `events_staged` table vs the expected.
*  Once complete, it runs the GE test suite to verify the outputted derived tables vs the expected derived tables.
* The expected tables were generated using BQ web model with [this](https://github.com/snowplow/data-models/commit/d4fe03b5464ba1226138cc5bde6dfcc847117c30) commit included. This should make the output deterministic.
* The tests at present will fail due to the inclusion of the 'stray page pings' edge case, which the model currently doesn't handle deterministically. I am undecided whether to include this or not. It seems like something we should be aiming to solve and therefore should be testing for but equally having the tests fail continuously due to this isnt ideal.

### CODE REVIEW

File: .scripts/integration_test.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  bash $script_path/run_test.sh -m $MODEL -d $DATABASE -c events_staged_integration_test_${i} || exit 1;
+
+  bash .scripts/run_config.sh -b sql-runner -c $config_dir/post_test.json -t $script_path/templates/$DATABASE.yml.tmpl -v .test/integration_tests/$MODEL/v1/${DATABASE}_variables.yml.tmpl || exit;
+
+  echo "integration_check: run $i done";
+
+done || exit 1
+
+echo "integration_check: Checking actual vs. expected for derived tables";
```
Comment (colmsnowplow) [inline .scripts/integration_test.sh:46]: Checking the assumption - the intent is for this to run once, after all runs of the model are complete yeah?

**Review by colmsnowplow — APPROVED** [review]
LGTM!

File: .scripts/integration_test.sh (hunk 1 above)
Comment (bill-warner) [reply .scripts/integration_test.sh:46]: Yeh exactly
